extension-style/api/extension-style.api

```diff
@@ -3725,7 +3725,6 @@ public final class com/mapbox/maps/extension/style/sources/generated/GeoJsonSour
 }
 
 public final class com/mapbox/maps/extension/style/sources/generated/GeoJsonSource$Companion {
-	public final fun directSetterEnabled ()Z
```

Comment:
seems kotlin binary compatibility validator doesn't take the VisibleForTesting into consideration.